src/main/java/com/github/mc1arke/sonarqube/plugin/CommunityBranchPlugin.java

```diff
@@ -67,12 +73,106 @@ public void load(CoreExtension.Context context) {
                 //the name and description shown on the UI are automatically loaded from core.properties so don't need to be specified here
                 PropertyDefinition.builder(CoreProperties.LONG_LIVED_BRANCHES_REGEX).onQualifiers(Qualifiers.PROJECT)
                         .category(CoreProperties.CATEGORY_GENERAL).subCategory(CoreProperties.SUBCATEGORY_BRANCHES)
-                        .defaultValue(CommunityBranchConfigurationLoader.DEFAULT_BRANCH_REGEX).build());
+                        .defaultValue(CommunityBranchConfigurationLoader.DEFAULT_BRANCH_REGEX).build(),
+
+                PropertyDefinition.builder("sonar.pullrequest.provider")
+                        .category(PULL_REQUEST_CATEGORY_LABEL)
+                        .subCategory(GENERAL)
+                        .onlyOnQualifiers(Qualifiers.PROJECT)
+                        .name("Provider")
+                        .type(PropertyType.SINGLE_SELECT_LIST)
+                        .options("Github", "BitbucketServer", "BitbucketCloud").build(),
```

Comment:
`Github` isn't provided under this Pull Request so probably shouldn't be mentioned (I'll need to sort that out when we come to merge). You also don't have a `BitbucketCloud` provider, so is that supposed to be included?